test/test_multiphysics.py

```diff
@@ -558,10 +544,98 @@ def test_thermally_coupled_fluid_wall_with_radiation(
     fluid_rhs = fluid_rhs.energy
     solid_rhs = wall_energy_rhs/(wall_cp*wall_rho)
 
+    # FIXME check convergence instead?
     assert actx.to_numpy(op.norm(dcoll, fluid_rhs, np.inf)) < 1e-4
     assert actx.to_numpy(op.norm(dcoll, solid_rhs, np.inf)) < 1e-4
 
 
+@pytest.mark.parametrize("order", [1, 3])
```

Comment:
Maybe set `order = 3` inside the test instead of parametrizing? (Probably doesn't depend on order?)

Reply:
https://github.com/illinois-ceesd/mirgecom/pull/1009/commits/27c53617533dd08900f46a4f7958e486e41343c0